firmware/config/engines/me7pnp.cpp

```diff
@@ -126,10 +126,6 @@ void vag_18_Turbo() {
 
 	//ETB Settings
 
-#if EFI_FSIO
-	// todo lua setFsio (14, GPIOF_13, "1");
-	// todo lua setFsioExt (3, GPIOE_0, "0.15 90 coolant 120 min max 90 - 30 / 0.8 * +", 25);
```

Comment:
these comments have value please keep those

Reply:
this file does not have value

Reply:
let's move that single line to b6 config to note that radiator GPPWM could be lua

Reply:
as for file yes we can remove under separate PR

Reply:
https://github.com/rusefi/rusefi/pull/3878